documentation/src/docs/pravega-concepts.md

```diff
@@ -11,429 +11,312 @@ You may obtain a copy of the License at
-change.  However at time t1, the system noted an increase in the ingestion rate
-and chose to split Segment 1 into two parts.  We call this a Scale-up event.
- Before t1, Events with a Routing Key that hashes to the upper part of the key
-space (values 200-399) would be placed in Segment 1 and those that hash into the
-lower part of the key space (values 0-199) would be placed in Segment 0. After
-t1, Segment 1 is split into Segment 2 and Segment 3.  Segment 1 is sealed and it
-no longer accepts writes.  At this point in time, Events with Routing Key
-300 and above are written to Segment 3 and those between 200 and 299 would be
-written into Segment 2.  Segment 0 still keeps accepting the same range of
-Events as before t1.  
-
-We also see another Scale-up event at time t2, as Segment 0’s range of Routing
-Key is split into Segment 5 & Segment 4.  Also at this time, Segment 0 is sealed
-off so that it accepts no further writes.
-
-Segments covering a contiguous range of the key space can also be merged.  At
-time t3, Segment 2’s range and Segment 5’s range are merged into Segment 6 to
-accommodate a decrease in load on the Stream.
-
-When a Stream is created, it is configured with an Scaling Policy that
-determines how a Stream reacts to changes in its load.  Currently there are
-three kinds of Scaling Policy:
-
-1.  Fixed.  The number of Stream Segments does not vary with load
-
-2.  Size-based.  As the number of bytes of data per second written to the Stream
-    increases past a certain target rate, the number of Stream Segments is
-    increased.  If it falls below a certain level, decrease the number of Stream
-    Segments.
-
-3.  Event-based.  Similar to Size-based Scaling Policy, except it uses the
-    number of Events instead of the number of bytes.
+- A Stream starts at time **_t0_** with a configurable number of Segments.  If the
+rate of data written to the Stream is constant, there will be no change in the number of Segments. 
+
+- At time **_t1_**, the system noted an increase in the ingestion rate and splits **_Segment 1_** into two parts. This process is referred as **Scale-up** Event.
+
+- Before **_t1_**, Events with a Routing Key that hashes to the upper part of the key
+space (i.e., values ranging from **200-399**) would be placed in **_Segment 1_** and those that hash into the
+lower part of the key space (i.e., values ranging from **0-199**) would be placed in **_Segment 0_**.
+
+- After **_t1_**, **_Segment 1_** is split into **_Segment 2_** and **_Segment 3_**. The **_Segment 1_** is sealed and stops accepting writes.  At this point in time, Events with Routing Key **_300_** and _above_ are written to **_Segment 3_** and those between **_200_** and **_299_** would be written into **_Segment 2_**.
+
+- **_Segment 0_** continues accepting the same range of Events as before **_t1_**.  
+
+- Another scale-up Event occurs at time **_t2_**, as **_Segment 0_**’s range of Routing
```

Comment:
Here `scale-up` is not in caps as the first mention to it. We need to unify style here.

Reply:
In line 114 , first occurrence, capitalized